pages/linux/ego.md

```diff
@@ -0,0 +1,32 @@
+# ego
+
+> Funtoo's official system personality management tool.
+> More information: <https://www.funtoo.org/Package:Ego>.
+
+- Synchronize the Portage tree:
+
+`ego sync`
+
+- Update bootloader configuration:
+
+`ego boot update`
+
+- Read a Funtoo wiki page:
+
+`ego doc {{wiki_page}}`
+
+- Print current profile:
+
+`ego profile show`
+
+- Enable a mix-in:
+
+`ego profile mix-in +{{mixin}}`
```

Comment:
Please briefly explain what is a `mix-in`. Also you can use a concrete example (such as `gnome`) to show what will be a proper argument of `mix-in`.
```suggestion
- Enable a mix-in:

`ego profile mix-in +{{gnome}}`
```